networking/networking-dashboards.adoc

```diff
@@ -21,3 +21,15 @@ You can view both general networking metrics as well as OVN-Kubernetes metrics f
 To view general networking metrics, select *Networking/Linux Subsystem Stats* from the *Dashboards* drop-down list. You can view the following networking metrics from the dashboard: *Network Utilisation*, *Network Saturation*, and *Network Errors*.
 
 To view OVN-Kubernetes metrics select *Networking/Infrastructure* from the *Dashboards* drop-down list. You can view the following OVN-Kuberenetes metrics: *Networking Configuration*, *TCP Latency Probes*, *Control Plane Resources*, and *Worker Resources*.
+
+[id="ingress-dashboards"]
+== Ingress Operator dashboard
+You can view networking metrics handled by the Ingress Operator from the dashboard. This includes things like the following:
```

Comment:
Suggestion: "things" reads just a bit informal and/or unspecific, maybe you can use a more specific word such as "metrics" even if it does repeat from the previous sentence:
```suggestion
You can view networking metrics handled by the Ingress Operator from the dashboard. This includes metrics like the following:
```

Up to you though!

Reply:
good suggestion, thanks!